src/main/java/uk/gov/hmcts/darts/event/service/impl/EventServiceImpl.java

```diff
@@ -34,4 +41,19 @@ public EventEntity getEventByEveId(Integer eveId) {
     public EventEntity saveEvent(EventEntity eventEntity) {
         return eventRepository.save(eventEntity);
     }
+
+    @Override
+    public List<EventEntity> getAllCourtCaseEventVersions(CourtCaseEntity courtCase) {
+        List<EventEntity> allEvents = new ArrayList<>();
```

Comment:
We select the events from two separate places this is just to ensure that if the same event is in both locations we we ignore the second one

Reply:
Yes, you can get duplicates but these should be cleaned up every evening as part of the [Duplicated event cleanup task ](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1689786831#TaskautomationLLD-Removeduplicatedeventstask)

Reply:
For the example above, it is highly unlikely you would get duplicates because it would need to be the same event that was received once into legacy and then once into modernised. This is an extremely unlikely edge case. 
However, there is probably no harm in making this a set as suggested, just in case @mario-paniccia ?